x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -198,6 +198,9 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
         },
       },
     },
+    revision: {
+      type: 'long',
```

Comment:
can we put `index:false`?

Reply:
Talking to you, if you need this field to be queryable, can you just add it and set it to true because we are going to have an exercise for 8.8 to define all these attributes with an index.

Reply:
Sounds good -- I've added `index:true` and left an explicit comment for our future selves as well 🙂